readme.md

```diff
@@ -691,7 +696,70 @@ coordsToPair(pairToCoords([1, 2])) // [1, 2]
 pairToCoords(coordsToPair({x: 1, y: 2})) // {x: 1, y: 2}
 ```
 
+### Catamorphism
 
+A `reduceRight` function. Take a bunch of things, and combine them into another. The morphism is from "bunch of things" to "another".
```

Comment:
Quite right. In the most general case, it applies to any `Foldable` structure. `bunch of things` could indeed be `thing` or even `no things`!